ConvivaTestApp/src/androidTest/java/com/bitmovin/analytics/conviva/testapp/TestBase.kt

```diff
@@ -33,19 +34,19 @@ open class TestBase {
     ) as HashMap<String, Any>
     val CUSTOM_ERROR_MESSAGE = "CUSTOM_ERROR_MESSAGE"
 
-    val DEFAULT_DASH_VOD_SOURCE = SourceItem(DASHSource("https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd"))
+    val DEFAULT_DASH_VOD_SOURCE = Source.create(SourceConfig("https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd", SourceType.Dash))
```

Comment:
Cool that you are using `Source` here. For consistency I'd suggest either using `SourceConfig` here or also using `Source` in business logic above. Basically it doesn't matter 😉 just another nitpick.

Reply:
@jarhoax , can you please elaborate on this comment?

Reply:
Here you are using the new `Source` type (which has been introduced with v3), which wraps a `SourceConfig`. The latter has been there since v2 and has been updated to make migration as easy as possible. Somewhere in this PR (I couldn't find it anymore 👀 ) I thought you were operating/instantiating `SourceConfig`s instead of `Source`. While both is totally fine, I just asked for consistency here, so either go with `SouceConfig` (only) also in those tests or use `Source` in business logic as well. 

As I cannot find the place I was referring to initially, feel free to ignore the initial comment. As said anyways just a nitpick, and not a necessary change.